README.md

```diff
@@ -95,11 +82,9 @@ Content-Type: application/json
 }
 ```
 
-### /api/migrate: Migration From Mandrill To SparkPost
-
-Extract a template from Mandrill, translate it and import it into SparkPost.
+#### /api/migrate: Migration From Mandrill To SparkPost
 
-Note: your SparkPost API key must include Templates Read/Write privileges for use with the `migrate` endpoint and the migration UI.
+Extracts a template from Mandrill, translates it and imports it into SparkPost. Your SparkPost API key must include `Templates Read/Write` permission in order to add templates to your account. Manage your API keys from the [API Keys page](https://app.sparkpost.com/account/api-keys)([EU](https://app.eu.sparkpost.com/account/api-keys)).
```

Comment:
We don't have EU support. I suggest either we add a control and API field for it or we remove this and add a caveat to the top of README.